### PR TITLE
Create `ImageCSVDataset` class, add script for loading .pkl files

### PR DESCRIPTION
**Pull request by Lakshita110**
Reason: Previous version of data loader was not appropriate for our purposes. 

Fix: Create a new `ImageCSVDataset` class which inherits from torchvision VisionDataset. This makes a better foundation for working with PyTorch's DataLoader, batching, and data transformations and makes it easy to integrate into PyTorch's training and evaluation pipelines. 

Verification: In individal comments in the files.

### CODE REVIEW

**Comment by CynthiaFeeney** [comment]
I added some commits to fix code formatting and improve the new dataset class.

Additional suggestions for @Lakshita110 :

- [x] Add skimage to the Pipenv ([instructions](https://pipenv.pypa.io/en/latest/installation.html#installing-packages-for-your-project))
- [x] This has not been communicated in our meetings, but pull requests should document why they're being made, what's changed, and verification the solution works. See an [example pull request](https://github.com/tufts-ml/hugheslab-onboarding/pull/4) for how to format this.

File: src/load_experiment.py
Comment (Lakshita110) [inline src/load_experiment.py:1]: This script was used to produce the following plots 
Experiment201

File: src/dataset_csv.py
Comment (Lakshita110) [inline src/dataset_csv.py:1]: In a downstream branch `retina-data` this DataLoader was used to load the IDRiD dataset and produce the following plot to check the samples. 
image

**Comment by CynthiaFeeney** [comment]
Is there a reason the labels are floats and not ints? The latter is standard.

**Comment by Lakshita110** [comment]
> Is there a reason the labels are floats and not ints? The latter is standard.

Fixed. I believe ints make sense.

**Comment by Lakshita110** [comment]
@PatrickFeeney Could you take a look at this and merge into v2?

**Comment by CynthiaFeeney** [comment]
The code is pretty good, just needs to be cleaned up a bit:

- [x] Install the [Flake8 extension](https://marketplace.visualstudio.com/items?itemName=ms-python.flake8) and use its auto-formatter (Shift + Alt + F) to clean up spacing issues.
- [x] Install the [autoDocstring extension](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) and fill out the docstring template I added in the latest commit. The template was produced by entering `"""` under the function and pressing Enter. The VisionDataset docstring can be used to describe the variables besides `csv_file`.

**Comment by Lakshita110** [comment]
Both things mentioned above should be fixed now.

**Comment by CynthiaFeeney** [comment]
Looks like the ImageCSVDataset docstring is missing, but should be able to be merged after that.

**Comment by Lakshita110** [comment]
> Looks like the ImageCSVDataset docstring is missing, but should be able to be merged after that.

Do you mean a docstring for the class itself?
